src/sentry/static/sentry/app/views/settings/project/projectServiceHooks.tsx

```diff
@@ -17,61 +17,61 @@ import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader
 import Switch from 'app/components/switch';
 import Truncate from 'app/components/truncate';
 import {IconAdd, IconFlag} from 'app/icons';
+import {ServiceHook} from 'app/types';
 
-class ServiceHookRow extends React.Component {
-  static propTypes = {
-    orgId: PropTypes.string.isRequired,
-    projectId: PropTypes.string.isRequired,
-    hook: PropTypes.object.isRequired,
-    onToggleActive: PropTypes.func.isRequired,
-  };
+type RowProps = {
+  orgId: string;
+  projectId: string;
+  hook: ServiceHook;
+  onToggleActive: () => void;
+};
 
-  state = {
-    loading: false,
-    error: false,
-  };
-
-  render() {
-    const {orgId, projectId, hook} = this.props;
-    return (
-      <Field
-        label={
-          <Link to={`/settings/${orgId}/projects/${projectId}/hooks/${hook.id}/`}>
-            <Truncate value={hook.url} />
-          </Link>
-        }
-        help={
-          hook.events && hook.events.length !== 0 ? (
-            <small>{hook.events.join(', ')}</small>
-          ) : (
-            <small>
-              <em>no events configured</em>
-            </small>
-          )
-        }
-      >
-        <Switch
-          isActive={hook.status === 'active'}
-          size="lg"
-          toggle={this.props.onToggleActive}
-        />
-      </Field>
-    );
-  }
+function ServiceHookRow({orgId, projectId, hook, onToggleActive}: RowProps) {
+  return (
+    <Field
+      label={
+        <Link to={`/settings/${orgId}/projects/${projectId}/hooks/${hook.id}/`}>
+          <Truncate value={hook.url} />
+        </Link>
+      }
+      help={
+        hook.events && hook.events.length !== 0 ? (
+          <small>{hook.events.join(', ')}</small>
+        ) : (
+          <small>
+            <em>no events configured</em>
+          </small>
```

Comment:
```suggestion
          <small>{hook.events && hook.events.length !== 0 ? hook.events.join(', ') : <em>{t('no events configured')}</em>}</small>
```

Reply:
Thanks! I applied this locally as prettier would get cranky.